python/pacemaker/_cts/tests/quorumfailure.py

```diff
@@ -0,0 +1,56 @@
+""" Test-specific classes for Pacemaker's Cluster Test Suite (CTS)
+"""
+
+__all__ = ["QuorumFailure"]
+__copyright__ = "Copyright 2000-2023 the Pacemaker project contributors"
+__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+
+from pacemaker import BuildOptions
+from pacemaker._cts.tests.ctstest import CTSTest
+from pacemaker._cts.tests.restarttest import RestartTest
+from pacemaker._cts.tests.simulstartlite import SimulStartLite
+from pacemaker._cts.tests.simulstoplite import SimulStopLite
+
+
+class QuorumFailure(CTSTest):
+    '''Set up a custom test to cause quorum failure issues for Andrew'''
```

Comment:
Despite this comment, a better name for this test would be something like ResyncCIB. It wipes the CIB from one node, starts it (without quorum), then ensures it picks it up correctly from the other nodes when they join.

Reply:
Fixed.